src/modules/ekf2/EKF2.cpp

```diff
@@ -582,8 +582,7 @@ void EKF2::PublishInnovations(const hrt_abstime &timestamp, const imuSample &imu
 	_ekf.getHaglInnov(innovations.hagl);
 	// Not yet supported
 	innovations.aux_vvel = NAN;
-	innovations.fake_hpos[0] = innovations.fake_hpos[1] = innovations.fake_vpos = NAN;
```

Comment:
We're currently publishing fake position innovations using the GPS fields, we should change that at some point. But we can re-add them when needed.